liblangutil/Token.h

```diff
@@ -215,12 +217,12 @@ namespace solidity::langutil
 	K(Bool, "bool", 0)                                                 \
 	K(Fixed, "fixed", 0)                                               \
 	K(UFixed, "ufixed", 0)                                             \
-	T(IntM, "intM", 0)                                                 \
-	T(UIntM, "uintM", 0)                                               \
-	T(BytesM, "bytesM", 0)                                             \
-	T(FixedMxN, "fixedMxN", 0)                                         \
-	T(UFixedMxN, "ufixedMxN", 0)                                       \
-	T(TypesEnd, nullptr, 0) /* used as type enum end marker */         \
+	K(IntM, "intM", 0)                                                 \
```

Comment:
I think this would mean `intM` is an invalid symbol from now?

Reply:
ah, that's why!